### ISSUE TITLE
Missing an icon (seo-line)?

### ISSUE THREAD

**Issue by Liam26** [bayesimpact/remixicon-react#158]
Hey, I'm currently trying to use the package for remix icons and it's been great, i've just recently run into a small problem. 

I searched https://remixicon.com/ for the icon `seo` and found this one https://remixicon.com/icon/seo-line

I'd like to use it in my app but when I try to translate it into Pascal case and use it like this:

`import SearchIcon from 'remixicon-react/SeoLineIcon';`
I also tried
`import SearchIcon from 'remixicon-react/SEOLineIcon';`

I just get the following error:

`Cannot find module 'remixicon-react/SeoLineIcon' or its corresponding type declarations.`

Is this something I am doing wrong with the import or is this icon missing from the package?

Thanks!

**Comment by EliotSlevin** [comment]
@Liam26 I don't think this package is maintained, I've also noticed newer icons not being available. 

It turns out the official package is `@remixicon/react` now, not `remix-icons/react`. The official package has all of the icons. I suspect this package was made before remix had their own react package?

**Comment by EliotSlevin** [comment]
https://github.com/bayesimpact/remixicon-react/issues/159

**Comment by Liam26** [comment]
> [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/u/40142244?s=64&amp;u=dbf904c4f62f21b6991d2f565b9ad1793ea4f7e9&amp;v=4">@Liam26](https://github.com/Liam26) I don't think this package is maintained, I've also noticed newer icons not being available.
> 
> It turns out the official package is `@remixicon/react` now, not `remix-icons/react`. The official package has all of the icons. I suspect this package was made before remix had their own react package?

Weirdly i've managed to find this exactly a year later just by chance as I missed this the first time! TBH we swapped to using a different package anyway and I think we will move over to using the official react package. Don't see this one getting updated anymore. 

Hope you managed to sort your issues with least pain as possible!
